src/elements/widgets/designerView/extensions/contextMenu/MultipleItemsSelectedContextMenu.ts

```diff
@@ -7,21 +8,57 @@ export class MultipleItemsSelectedContextMenu implements IContextMenuExtension {
 
   public orderIndex: number = 60;
 
-  public shouldProvideContextmenu(event: MouseEvent, designerView: IDesignerCanvas, designItem: IDesignItem, initiator: ContextmenuInitiator) {
+  public shouldProvideContextmenu(event: MouseEvent, designerCanvas: IDesignerCanvas, designItem: IDesignItem, initiator: ContextmenuInitiator) {
     if (designItem.instanceServiceContainer.selectionService.selectedElements.length > 1) {
       return true;
     }
     return false;
   }
 
-  public provideContextMenuItems(event: MouseEvent, designerView: IDesignerCanvas, designItem: IDesignItem): IContextMenuItem[] {
+  public provideContextMenuItems(event: MouseEvent, designerCanvas: IDesignerCanvas, designItem: IDesignItem): IContextMenuItem[] {
     return [
       {
         title: 'wrap in',
         children: [
           {
             title: 'div',
-            action: () => { }
+            action: () => {
+              const grp = designItem.openGroup("wrap in Div");
+              let elements = designItem.instanceServiceContainer.selectionService.selectedElements;
+              let div = document.createElement('div');
+              const divDesignItem = DesignItem.createDesignItemFromInstance(div, designItem.serviceContainer, designItem.instanceServiceContainer);
+              designItem.insertAdjacentElement(divDesignItem, 'beforebegin');
+              let offset = 10;
+              let minX = Number.MAX_VALUE;
+              let minY = Number.MAX_VALUE;
+              let maxX = 0;
+              let maxY = 0;
+              for (let e of elements) {
+                let rect = designerCanvas.getNormalizedElementCoordinates(e.element);
+                if (rect.x < minX)

```

Comment:
hier kann man sich in der Zukunft mal überlegen, ob man das nicht irgendwo in eine statische utillity Klasse packt. Das berechnet ja quasi nur die maximum bounds, das wird man mit hoher Wahrscheinlichkeit öfter brauchen und so muss man es dann nur an einer Stelle pflegen.

Muss von mir aus nicht jetzt sein, kann man sich aber mal überlegen

Reply:
LGTM

Reply:
hatte ich heute ausch schon gesagt.
wir werden das irgendwann in eine hilfsklasse packen. man muss ja auch noch berücksichtigen wie ein element positioniert ist top oder bottom,...